### PR TITLE
docs(compute-samples): added snippets and tests for Windows instance 

### PR DESCRIPTION
**Pull request by Sita04**
Added snippets for Windows Instance management

### CODE REVIEW

File: compute/cloud-client/src/main/java/compute/windows/windowsinstances/CreateFirewallRuleForWindowsActivationHost.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    createFirewallRuleForWindowsActivationHost(projectId, firewallRuleName, networkName);
+  }
+
+  // Creates a new allow egress firewall rule with the highest priority for host
+  // kms.windows.googlecloud.com (35.190.247.13) for Windows activation.
+  public static void createFirewallRuleForWindowsActivationHost(String projectId,
+      String firewallRuleName, String networkName)
+      throws IOException, ExecutionException, InterruptedException {
+
+    try (FirewallsClient firewallsClient = FirewallsClient.create()) {
```
Comment (averikitsch) [inline compute/cloud-client/src/main/java/compute/windows/windowsinstances/CreateFirewallRuleForWindowsActivationHost.java:52]: See go/cdpe-snippet-rubric for additional comments that should be added

File: compute/cloud-client/src/main/java/compute/windows/windowsinstances/CreateFirewallRuleForWindowsActivationHost.java [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+      Firewall firewall = Firewall.newBuilder()
+          .setName(firewallRuleName)
+          .addAllowed(Allowed.newBuilder()
+              .setIPProtocol("tcp")
+              .addPorts("1688")
```
Comment (averikitsch) [inline compute/cloud-client/src/main/java/compute/windows/windowsinstances/CreateFirewallRuleForWindowsActivationHost.java:58]: Will users know why these are predefined?

Comment (Sita04) [reply compute/cloud-client/src/main/java/compute/windows/windowsinstances/CreateFirewallRuleForWindowsActivationHost.java:58]: These are the default values for kms.windows.googlecloud.com 
https://cloud.google.com/compute/docs/instances/windows/creating-managing-windows-instances#firewall_rule_requirements

I'll add a comment there referencing this doc. Thanks.

**Comment by Sita04** [comment]
Blocked by https://github.com/GoogleCloudPlatform/java-docs-samples/pull/7071

**Review by averikitsch — APPROVED** [review]

**Pull request closed without merging**